crates/rattler-bin/src/commands/create.rs

```diff
@@ -60,12 +60,23 @@ pub struct Opt {
 
     #[clap(long)]
     timeout: Option<u64>,
+
+    #[clap(long)]
+    target_prefix: Option<PathBuf>,
 }
 
 pub async fn create(opt: Opt) -> anyhow::Result<()> {
     let channel_config = ChannelConfig::default();
-    let target_prefix = env::current_dir()?.join(".prefix");
-
+    let target_prefix = opt.target_prefix.unwrap_or_else(|| {
+        env::current_dir()
+            .expect("could not find current dir")
+            .join(".prefix")
+    });
+    // First create the target prefix if it doesn't exist yet.
+    fs::create_dir_all(&target_prefix).expect("could not create target prefix");
```

Comment:
I think this should result in a user error instead of an expect.

The canonicalize could be an expect because we already created the directory.